### ISSUE TITLE
Add Carthage support

### ISSUE THREAD

**Issue by Vkt0r** [Vkt0r/AccordionSwift#36]

**Comment by DewaldDeJager** [comment]
@Vkt0r I have never used Carthage before, but may investigate this and possibly implement it, please? I think Swift Package Manager might also be a good idea.

**Comment by Vkt0r** [comment]
@DewaldDeJager Yes sure, we should not have any problem with Carthage as I tested recently. Pretty much the target should be set to `Shared` and we need to test the `carthage build --no-skip-current`. Feel free to read more [here](https://github.com/Carthage/Carthage#supporting-carthage-for-your-framework) and open a PR with updates. Thanks for you continued contribution !

**Comment by DewaldDeJager** [comment]
@Vkt0r Sorry for not getting to this yet. I saw you did it. We can close this issue now.
